### PR TITLE
Plotly ws/instructions update

### PR DESCRIPTION
**Pull request by euronion**
Hi there,

I updated the `environment.yml` file and streamlined the plotly workshop instructions a bit to only use a `conda` environment which automatically installs all dependencies. Hoping this makes entry for participants less familiar with Python/Anaconda/pip easier.

### CODE REVIEW

File: plotly-dash/README.md [hunk 1]
```diff
@@ -1,62 +1,103 @@
...
+```
 
-Note your environnment name here is `workshop_env` but you could change it to another name. If you do you need to replace `workshop_env` by your other name in the following commands
+2. Clone the repository
+```
+    > git clone https://github.com/rl-institut/workshop.git
+```
 
--you should be able to see it with the following command
+3. Move into the freshly cloned repositor
```
Comment (Bachibouzouk) [inline plotly-dash/README.md]:
```suggestion
3. Move into the freshly cloned repository
```

File: plotly-dash/environment.yml [hunk 2]
```diff
@@ -1,5 +1,9 @@
-name: myvenv
+name: plotly_workshop
+channels:
+  - conda-forge
+  - defaults
 dependencies:
-- python=3.6
-- pip:
-   - -r requirements.txt
+  - pandas
```
Comment (Bachibouzouk) [inline plotly-dash/environment.yml]: Do you have an argument not to use a link to the `requirements.txt` file? My argument is that I prefer to have only one place where I define the dependencies

Comment (euronion) [reply plotly-dash/environment.yml]: I forgot to delete the `requirements.txt` from the branch.
The idea was to only use ` conda` with the `environment.yml` file and remove the `requirements.txt` alltogether.

Comment (Bachibouzouk) [reply plotly-dash/environment.yml]: The solution with `requirements.txt` linked in `environment.yml` works both for conda user and for non-conda users (like me), this is why I had implemented it. If `requirements.txt` is there it should not bother the beginners too much as they would only follow the instructions.

Comment (euronion) [reply plotly-dash/environment.yml]: Ah, gotcha. Shall I restore the original version with conda and pip?

Comment (Bachibouzouk) [reply plotly-dash/environment.yml]: If you don't mind yes, otherwise I can do it :)

**Comment by euronion** [comment]
Done. I removed the `python=3.6` version shakle, added `pip` as explicit dependency for `conda` (otherwise it is annoying about `pip` not being an explicit dependency) and made the environment names coherent between the readme and the conda environment file. (I used your `workshop_env` name for the environment again).

**Pull request merged**